Bottom: 8fd8c8fccf5c5b9308b66e1231c54c8fabd5567e
Top:    92c393f9bfbbc72c234dca9a72febe2f88c6dee1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-05 21:05:52 +0100

Crasy test


---

diff --git a/src/draw.c b/src/draw.c
index c8599fd..513bc47 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -141,10 +141,12 @@ set_pv_color (PinType *pv, int type)
 }
 
 static int
-pad_callback (const BoxType * b, void *cl)
+pad_inlayer_callback (const BoxType * b, void *cl)
 {
   PadTypePtr pad = (PadTypePtr) b;
-  int *side = cl;
+  LayerType *layer = cl;
+  int solder_group = GetLayerGroupNumberByNumber (solder_silk_layer);
+  int group = GetLayerGroupNumberByPointer (layer);
 
   if (ON_SIDE (pad, *side))
     {
